### PR TITLE
Rebuild for gfortran 7 for OSX (fixed)

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes #39

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin, please rerender

**Comment by beckermr** [comment]
@conda-forge-admin, please rerender

**Comment by beckermr** [comment]
@conda-forge/esmf This PR is ready to merge!

**Comment by ocefpaf** [comment]
Thanks @beckermr and @isuruf for taking care of this one!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,12 +21,12 @@ requirements:
...
   host:
     - netcdf-fortran
-    - libnetcdf
+    - libnetcdf >=4.7
     - hdf5
     - mpich
   run:
     - netcdf-fortran
-    - libnetcdf
+    - libnetcdf >=4.7
```
Comment (ocefpaf) [inline recipe/meta.yaml]: The whole stack is not using `libnetcdf >=4.7` yet. That is fine here but we should probably start thinking about a migration.

**Pull request merged**